test/OpenJ9_Jsr_292_API/src/com/ibm/j9/jsr292/LookupInTests.java

```diff
@@ -0,0 +1,616 @@
+/*******************************************************************************
```

Comment:
Why are there two LookupInTests.java source files?  Are the tests the same?  Different?

Reply:
We need to call a few methods of Module (e.g. isExported) which are missing in the current build with b148. So the easiest way for us is to ensure the new code only works on openj9 (e.g. b175) rather then the current build (b148).

To get all existing cases work on Java 8/b148,  all these tests in the original LookupInTests.java plus the required files were copied to a new test project called OpenJ9_Jsr_292_API as suggested by the FVT team. 

As a result, test/Jsr292/src/com/ibm/j9/jsr292/LookupInTests.java only work for Java 8 (829) and Java 9 with b148, while test/OpenJ9_Jsr_292_API/src/com/ibm/j9/jsr292/LookupInTests.java only work on openJ9.

The only difference of these two files is that all Module related test cases in OpenJ9_Jsr_292_API were updated to support MODULE/UNCONDITIONAL bit.

Reply:
I thought that we had some other test examples where there were differences between 829 and OpenJ9, where we moved the common code out for reuse by both to minimize duplication (like in Java8andUp).  @renfeiw is there an opportunity to do similar with this code to avoid bloating the repo with lots of duplication?

Reply:
We don't have a better way for this case right now. If we want share the same file without referencing across java projects, we need to place the new tests into the existing Jsr292 project. However, the Jsr292 project is excluded from openj9 currently. So the added tests will be excluded as well. We are working on a better way to exclude test, with that in place we will be able to handle this case in the future.